src/main/features/claimant-response/routes/check-and-send.ts

```diff
@@ -50,7 +51,7 @@ export default express.Router()
             break
         }
       }
-
```

Comment:
This block of code from this file needs to be removed.
```
if (draft.document.formaliseRepaymentPlan && draft.document.formaliseRepaymentPlan.option) {
        switch (draft.document.formaliseRepaymentPlan.option) {
          case FormaliseRepaymentPlanOption.REQUEST_COUNTY_COURT_JUDGEMENT:
            await CCJClient.issue(claim, draft, user)
            break
          case FormaliseRepaymentPlanOption.SIGN_SETTLEMENT_AGREEMENT:
            const settlement: Settlement = prepareSettlement(claim, draft.document)
            await OfferClient.signSettlementAgreement(claim.externalId, user, settlement)
            break
        }
      }
```

Reply:
done I have removed this code.